examples/cloudml-sklearn-pipeline/trainer/task.py

```diff
@@ -75,8 +75,14 @@ def run_experiment(flags):
   """Testbed for running model training and evaluation."""
   # Get data for training and evaluation
 
-  dataset = utils.read_df_from_bigquery(
+  # If BigQuery table, specify as as PROJECT_ID.DATASET.TABLE_NAME
+  if len(flags.input.split('.')) == 3:
```

Comment:
You could consider handling the legacy sql table reference `project:dataset.table` which is what the `bq` cli still accepts.

Reply:
Hi @jaketf , thanks a lot for approving. I actually make the standard sql as the requirement while initiating the client.